src/logging.h

```diff
@@ -20,8 +21,14 @@
 #define JVM_ERR_FAIL_COND_V_MSG(condition, ret_var, message) \
     ERR_FAIL_COND_V_MSG(condition, ret_var, vformat("Godot-JVM: %s", message))
 
-#define JVM_CRASH_COND_MSG(condition, message) CRASH_COND_MSG(condition, vformat("Godot-JVM: %s", message))
-
-#define JVM_CRASH_NOW_MSG(message) CRASH_NOW_MSG(vformat("Godot-JVM: %s", message))
+#define JVM_CRASH_COND_MSG(condition, message) \
+    if (unlikely(condition)) { OS::get_singleton()->alert(message, "Fatal error"); } \
```

Comment:
Not a big fan of adding another if for crash cond.  
In this case this will crash, so crash message should be enough.  
Or this if can be debug only.

Reply:
the issue is that the crash message is only logged to the terminal. If you did not start the game from the terminal you'll never see it. But i agree, a ifdef for debug only would probably be best